frontend/packages/dev-console/src/components/import/devfile/DevfileImportForm.tsx

```diff
@@ -22,9 +27,36 @@ const DevfileImportForm: React.FC<FormikProps<FormikValues> & DevfileImportFormP
   setFieldValue,
 }) => {
   const { t } = useTranslation();
+  const searchParams = new URLSearchParams(window.location.search);
+  const gitRepoUrl = searchParams.get('gitRepo');
+  const devfileName = searchParams.get('devfileName') || detectGitRepoName(gitRepoUrl);
+  const formType = searchParams.get('formType');
+  const language = searchParams.get('language');
+  const selectedBuilderImage = builderImages[language];
```

Comment:
I think there's been some confusion here. Shouldn't we show selected devfile instead of selectedBuilderImage here?

Reply:
Sorry I was confused with builder image samples, updated to use the selected devfile sample.